website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -46,16 +46,21 @@ To create a new run task:
 
 1. Click the **+** next to the task you want to add to the workspace.
 
+1. Choose when to run the task:
+
+   - **Pre-plan**: Before the plan is generated.
+   - **Post-plan**: After the plan is generated.
```

Comment:
It would be great to possibly list 
```suggestion
   - **Pre-plan**: Before Terraform creates the plan.
   - **Post-plan**: After Terraform creates the plan.
```